controller/api/destination/watcher/endpoints_watcher.go

```diff
@@ -712,7 +712,7 @@ func (pp *portPublisher) endpointSliceToAddresses(es *discovery.EndpointSlice) A
 
 				identity := es.Annotations[consts.RemoteGatewayIdentity]
 				address, id := pp.newServiceRefAddress(resolvedPort, IPAddr, serviceID.Name, es.Namespace)
-				address.Identity, address.AuthorityOverride = authorityOverride, identity
+				address.Identity, address.AuthorityOverride = identity, authorityOverride
```

Comment:
oops 😅 I wonder if it isn't better to have this in two separate lines (instead of a one liner) so people like me don't accidentally mix them up?